src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search"
+      action="http://localhost:3000/api"
+      method="post"
+      >
+      <div class="search__bar__content" data-qa="big">
+        <img
+          class="icon big-icon"
+          src="images/Search.svg"
+          alt="search"
```

Comment:
Alt attribute is a very important attribute
Give a more clearly description to this attribute